depends/funcs.mk

```diff
@@ -173,6 +174,10 @@ endif
 endef
 
 define int_add_cmds
+ifneq ($(LOG),)
+$(1)_logging = >>$$($(1)_build_log) 2>&1 || { if test -f $$($(1)_build_log); then cat $$($(1)_build_log); fi; exit 1; }
```

Comment:
So this would mean that while the `$(1)_*_cmds` is running, we wouldn't see any logs, and only if the `cmds` fail, would we `cat` the logs? Am I understanding this correctly?

Reply:
Yes, you're correct. This behavior is applied only if when `LOG` is set.